setup.py

```diff
@@ -8,22 +8,20 @@
 import subprocess
 
 test_deps = [
-    'pytest>=3.6',
-    'pytest-cov==2.4.0',
-    'coveralls[yaml]==1.6.0',
-    'pytest-xdist==1.18.1',
-    'py-evm==0.2.0a42',
```

Comment:
Note `py-evm` was removed as a  pinned dependency, allowing `eth-tester` to specify